lisk/EXCEPTIONS.md

```diff
@@ -0,0 +1,231 @@
+## Exceptions
+
+During the development of Lisk Core, we found some edge cases when the existing data on the network becomes invalid. Either it was because of a bug or some protocol change. To maintain state of the chain we have to consider that particular data valid in that particular scope. This concept of making some invalid data to be valid in the network is known to us as exceptions. To see the full list of available exception categories see the following [exception schema](https://github.com/LiskHQ/lisk-sdk/blob/ca7a626f5d231f07ac784b4f6b58ed72dfa9aaa3/framework/src/modules/chain/defaults/config.js#L110)
+
+Below is the list of exceptions on each network with some details.
```

Comment:
This file will be hard to maintain as we will need to remember to update it every time we update the exceptions. I would rather add the details to the `schema` itself in the `description` attribute.

Reply:
I hope we don't have to maintain this much, so I think it should be fine, although documentation should be improved for readability in the future.

@MaciejBaj what do you think?

Reply:
Yes it seems hard to maintain for individuals, but exceptions are something special case in our application. And when we encounter any such exception, reviewer must keep track of it in this file or separate document.